doc/input-output-reference/src/overview/group-plant-equipment.tex

```diff
@@ -4962,214 +4964,342 @@ \subsubsection{Inputs}\label{inputs-15-010}
+
+\paragraph{Water to Water Heat Pump Source Side Mass Flow Rate {[}kg/s{]}}\label{water-to-water-heat-pump-source-side-mass-flow-rate-kgs-1}
+
+This output variable represents the average fluid flow rate through the source side coil. The values are calculated for each HVAC system time step being simulated, and the results are averaged for the time step being reported.
+
+\subsection{HeatPump:WaterToWater:EIR:Cooling (and Heating)}\label{wwhpeir}
+
+Next, the EIR-formulated model objects are described. Although the model follows the pattern of having a separate input object for cooling and heating, the inputs are essentially identical, so they are described together.  Note that this component is currently a constant flow device (it will always request its full design flow from the plant), with a future addition to make it respond to outlet setpoint in order to vary the flow request.
+
+\subsubsection{Inputs}\label{wwhpeir_inputs}
+
+\paragraph{Field: Name}\label{wwhpeir_inputs_name}
+
+This alpha field contains the identifying name for the ground source heat pump.
+
+\paragraph{Field: Load Side Inlet Node Name}\label{wwhpeir_inputs_load_inlet_node}
+
+This alpha field contains the heat pump's load side inlet node name.
+
+\paragraph{Field: Load Side Outlet Node Name}\label{wwhpeir_inputs_load_outlet_node}
+
+This alpha field contains the heat pump's load side outlet node name.
+
+\paragraph{Field: Source Side Inlet Node Name}\label{wwhpeir_inputs_source_inlet_node}
+
+This alpha field contains the heat pump's source side inlet node name.
+
+\paragraph{Field: Source Side Outlet Node Name}\label{wwhpeir_inputs_source_outlet_node}
+
+This alpha field contains the heat pump's source side outlet node name.
+
+\paragraph{Field: Companion Heat Pump Name}\label{wwhp_eir_inputs_companion_name}
+
+This optional alpha field contains the name of a companion heat pump object.  If the current object is heating type then this is expected to include the paired cooling object, and vice-versa.  The companion is used for sizing if plant sizing data is unavailable, and during the simulation to ensure that any unexpected simultaneous operation of the two objects provides a warning to the user.
+
+\paragraph{Field: Load Side Reference Flow Rate}\label{wwhp_eir_inputs_load_side_flow}
+
+This autosizable field defines the reference load side flow rate for the coil.  This field is used as the nominal flow request for this heat pump unit during the simulation.  The units for this field are [m3/s].
+
+\paragraph{Field: Source Side Reference Flow Rate}\label{wwhp_eir_inputs_source_side_flow}
+
+This autosizable field defines the reference demand side flow rate for the coil.  This field is used as the nominal flow request for this heat pump unit during the simulation.  The units for this field are [m3/s].
```

Comment:
. . . defined the reference ~demand~ *source* side flow rate . . . for the ~coil~*heat pump*.

Reply:
:ballot_box_with_check: